block.go

```diff
@@ -84,6 +84,9 @@ type IndexReader interface {
 	// LabelIndices returns a list of string tuples for which a label value index exists.
 	LabelIndices() ([][]string, error)
 
+	// GetSymbolTableSize returns the size occupied by the symbol table of Reader object.
+	GetSymbolTableSize() uint32
+
```

Comment:
This has nothing to do with functionality of an `IndexReader` and makes strong assumptions about how it is implemented. However we want to expose it, this interface isn't it.

Reply:
I had the same question in this https://github.com/prometheus/tsdb/pull/297 PR.
There I have added a new interface `IndexExtra` for exposing LabelName() function.
Does this way work or is there any other way to expose functions from index.go ?